src/util/tile-checker.js

```diff
@@ -0,0 +1,27 @@
+var $ = require('jquery');
+var getValue = require('./get-object-value');
+var limitsSVG = require('../../themes/img/timeout.svg');
```

Comment:
For me it is not a timeout svg or limits svg, for the moment is an error image, that's it.